### ISSUE TITLE
Add a note on nvda-old.log and crash dump files in the user guide

### ISSUE THREAD

**Issue by nvdaes** [nvaccess/nvda#5604]
NVDA should document where both nvda and nvda-old.log, as well as minidumps (for NVDA or specific applications) are stored and when they are useful, i.e. when reporting bugs.
These information is in the wiki, but it's not translated and people can use in different language. Therefore, these files are generated by NVDA and they are useful for every user, not intended just for developers. So, imo a note aboute this should be included in the user guide.
Thanks.

**Comment by jcsteh** [comment]
They're not useful for the majority of users, which is what the User Guide is intended for. They're only really of use to developers or users with reasonable technical knowledge, in which case the wiki is an acceptable place to find this information. Furthermore, you need to be able to read/write English in order to use this information or report bugs, so translating this doesn't make a huge amount of sense.

**Comment by nvdaes** [comment]
OK. Then I will post more information about these files on a spanish
blog, since sometimes people report bugs on the spanish mailing list.
Thanks.
